src/Theme/Page/Partner.elm

```diff
@@ -15,7 +15,13 @@ import Theme.TransMarkdown
 viewInfo localModel { partner, events } =
     section [ css [ margin2 (rem 0) (rem 0.35) ] ]
         [ text ""
-        , div [ css [ descriptionStyle ] ] (Theme.TransMarkdown.markdownToHtml (t (PartnerDescriptionText partner.description partner.name)))
+        , div [ css [ descriptionStyle ] ]
+            (Theme.TransMarkdown.markdownToHtml
+                (t (PartnerDescriptionText partner.summary partner.name)
+                    ++ "\n\n"
```

Comment:
I think instead of the manual newline between the 2 sections of text, we could copy the whole thing and make  2 divs - depends what styles we want to apply to both but we might want the `descriptionStyle` css to wrap both.

We also probably only want the "Ask partner for more information" to show for one or other. I guess leave it on the description? I think maybe this site was designed with the summary being used in the list and the description being used in this page so we might even want more logic to check if they repeat each other or only show empty text if both are empty?

It would be more clear if that logic to show empty text was here instead of in the translation file.

Try something like:
```
, div [ css [ descriptionStyle ] ] 
    [ div [ ] (Theme.TransMarkdown.markdownToHtml partner.summary)
    , div [ ] (Theme.TransMarkdown.markdownToHtml (t (PartnerDescriptionText partner.description partner.name)))
    ]
```
or
```
, div [ css [ descriptionStyle ] ] (Theme.TransMarkdown.markdownToHtml partner.summary)
, div [ css [ descriptionStyle ] ] (Theme.TransMarkdown.markdownToHtml (t (PartnerDescriptionText partner.description partner.name)))
```

Reply:
i tried this first but it adds awkward looking spacing between bc the div has padding on it too - i don't think we want the extra div?

ah i missed the no text state - i guess ideally this would be a fuction that processes the whole thing but getting a bit out my depth!

Reply:
We can fix the styling on the divs if there is too much padding - maybe we need to make a new style and not use the existing descriptionStyle.

You're doing fine with it! And yes, I think I'm inclined to change that copy key to `PartnerDecriptionEmptyText` and then as you say, make a function here that only shows it if both summary & description are empty. Shout if you want a hand.